client/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "outDir": "./dist/",
+    "noImplicitAny": true,
+    "module": "es6",
+    "target": "es5",
+    "jsx": "react",
+    "allowJs": true
+  }
```

Comment:
Other useful/recommended compiler options :

1. `skipLibCheck: true`: skips unnecessary lib checks in `*.d.ts` files (usually outside our scope)
2. `noEmit: true`: We use `webpack` so we should avoid emitting any output accidentally when running `tsc` manually.
3. `strict: true`: Enable strict type checking.
4. `forceConsistentCasingInFileNames: true`: Disallow inconsistently-cased references to the same file.